platforms/core-configuration/declarative-dsl-provider/src/testFixtures/groovy/org/gradle/internal/declarativedsl/settings/ProjectFeatureFixture.groovy

```diff
@@ -870,24 +859,19 @@ trait ProjectFeatureFixture extends ProjectTypeFixture {
                 import ${Definition.class.name};
                 import ${BuildModel.class.name};
                 import org.gradle.api.provider.Property;
-                import ${Restricted.class.name};
-                import ${Configuring.class.name};
                 import org.gradle.api.model.ObjectFactory;
                 import org.gradle.api.Action;
                 import org.gradle.api.tasks.Nested;
                 import javax.inject.Inject;
 
-                @${Restricted.class.simpleName}
                 public abstract class ${publicTypeClassName} implements ${Definition.class.simpleName}<${publicTypeClassName}.FeatureModel> {
-                    @${Restricted.class.simpleName}
                     public abstract Property<String> getText();
 
                     ${maybeInjectedServiceDeclaration}
 
                     @Nested
                     abstract Fizz getFizz();
 
-                    @${Configuring.class.simpleName}
```

Comment:
❓ Is `HiddenInDefinition` not required here?

Reply:
Ah, it was not required because `getFizz` was non-public.